shared_requirements.txt

```diff
@@ -225,6 +225,7 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-mixedreality-remoterendering azure-core<2.0.0,>=1.6.0
 #override azure-mixedreality-remoterendering azure-mixedreality-authentication>=1.0.0b1
 #override azure-ai-language-conversations azure-core<2.0.0,>=1.24.0
+#override azure-analytics-loadtestservice azure-core<2.0.0,>=1.23.0
```

Comment:
You could reference https://github.com/Azure/azure-sdk-for-python/wiki/How-to-fix-CI#problems-and-resolution to fix ci